src/front-end/typescript/lib/pages/proposal/code-with-us/edit/index.tsx

```diff
@@ -106,8 +106,8 @@ function makeInit<K extends Tab.TabId>(): component_.page.Init<
               proposalId,
               (response) => response
             ),
-            api.proposals.cwu.readOne(opportunityId)(
-              proposalId,
```

Comment:
There was another instance of duplicate proposal requests happening elsewhere in the code, I'm assuming that's also unintended and should be changed?

Reply:
I can't speak to whether it was intentional, but leaving it the way it is doesn't make sense. Would propose the following modification: 

```
 api.opportunities.cwu.readOne(
              opportunityId,
              (response) => response
            ),
```

Reply:
Yup, it was me, an unintended bug.

Reply:
> I can't speak to whether it was intentional, but leaving it the way it is doesn't make sense. Would propose the following modification:
> 
> ```
>  api.opportunities.cwu.readOne(
>               opportunityId,
>               (response) => response
>             ),
> ```

This ended one ended up being easy to miss, we weren't populating the evaluation criteria for admin/gov users:
<img width="640" alt="Screen Shot 2022-12-14 at 1 52 17 PM" src="https://user-images.githubusercontent.com/11410926/207723333-85a0f2c9-d008-40d1-a502-c578ca85bbca.png">